lib/index.js

```diff
@@ -51,10 +51,11 @@ function filterIgnoredFileMessages(errors) {
 function filterAllIgnoredFileMessages(result) {
   const resultOutput = result;
 
-  result.results.map((resultItem) => {
+  result.results.forEach((resultItem) => {
     resultItem.messages = filterIgnoredFileMessages(resultItem.messages);
     return resultItem;
```

Comment:
It looks like this is just a leftover from using `map`, but with `forEach`, the `return` actually becomes unnecessary 😄.